integration_tests/suite/base/test_incalls.py

```diff
@@ -165,6 +165,35 @@ def test_get(incall):
     )
 
 
+@fixtures.incall(id=0)
+@fixtures.user(id=1)
+@fixtures.incall(id=1)
+@fixtures.user(id=2)
+def test_get_userid(incall, user1, *_):
```

Comment:
nit: you mix get / list nomenclature for tests. This test should be bellow `test_sorting_offset_limit` and renamed to `test_list_by_user_id`

Reply:
Updated